app/API/Core/index.js

```diff
@@ -9,7 +9,7 @@ import Declarations from '../StyleSheet/Declarations';
 import includes from 'lodash/includes';
 
 function guessPlatform(): $reactors$platform {
-  if (typeof window !== 'undefined' && window.DOMError) {
+  if (typeof window !== 'undefined') {
```

Comment:
If I skip this check, mobile will then be web since mobile can have a window defined.